pkg/webui/components/navigation/side/item/index.js

```diff
@@ -53,7 +53,7 @@ const SideNavigationItem = props => {
         [],
       )
       for (const path of paths) {
-        if (location.pathname.startsWith(path)) {
+        if (location.pathname.endsWith(path)) {
```

Comment:
`paths` array looks like this:
`[
    "managed-gateway/connection-settings",
    "managed-gateway/wifi-profiles"
]`
and location.pathname is something like this: `/console/gateways/ttigpro/managed-gateway/connection-settings`

So logically the pathname should end with one of the paths. Previously if I reload the page the side menu item doesn't open.

Reply:
Hmm indeed. Something must've changed there at some point, because I'm pretty sure it used to work 🤷 
Possibly a regression from when we updated the router at some point.